solutions/cses-1736.mdx

```diff
@@ -0,0 +1,149 @@
+---
+id: cses-1736
+source: CSES
+title: Polynomial Queries
+author: Albert Ye
+---
+
+## Solution
+**Time Complexity: $\mathcal{O}(n\log n)$**
+
+In this question, we have range updates and range queries, but
+the updates are of a strange format: incrementing the first number
+in interval by $1$, the second by $2$, etc. However, despite the
```

Comment:
Instead of repeating the problem, something like "Despite the strange update format, we can still use..." is enough

Reply:
yep